pkg/signup/repository/postgresql/postgres.go

```diff
@@ -0,0 +1,54 @@
+package postgresql
+
+var db *sql.DB
+var err error
+
+//Insert user into DB 
```

Comment:
please update users.email with a unique constraint, and handle unique constraint violation exception